sktime/forecasting/model_selection/_split.py

```diff
@@ -472,8 +471,8 @@ def get_cutoffs(self, y: Optional[ACCEPTED_Y_TYPES] = None) -> np.ndarray:
 
         Returns
         -------
-        cutoffs : np.array
-            The array of cutoff points.
+        cutoffs : np.ndarray
+            The array of integer cutoff points.
```

Comment:
I would suggest to write:
1D np.ndarray of `int`
`iloc` location indices, in reference to `y`, of cutoff indices
(is that correct?)

Reply:
yes, totally agree. changed in all 3 palces